core/src/bytecode/typecheck/mod.rs

```diff
@@ -97,6 +97,27 @@ use self::subtyping::SubsumedBy;
 /// The max depth parameter used to limit the work performed when inferring the type of the stdlib.
 const INFER_RECORD_MAX_DEPTH: u8 = 4;
 
+//TODO[RFC007]: once we switch to this typechecker, replace this trait definition by an import of
+//`cache::AstImportResolver`.
+pub trait AstImportResolver<'ast> {
+    /// Resolve an import.
+    ///
+    /// Read and store the content of an import, put it in the file cache (or get it from there if
+    /// it is cached), then parse it and return the corresponding term and file id.
```

Comment:
This just returns the term, right? Not the file id

Reply:
Right, it's the old documentation that has been copy-pasted. Updated in latest commits.